packages/calypso-build/postcss.config.js

```diff
@@ -1,8 +1,7 @@
-module.exports = ( { options: { preserveCssCustomProperties = true } } ) => ( {
+module.exports = () => ( {
 	plugins: {
 		'postcss-custom-properties': {
-			importFrom: [ require.resolve( '@automattic/calypso-color-schemes' ) ],
-			preserve: preserveCssCustomProperties,
+			preserve: false,
```

Comment:
> The default value of `preserve` is and should be `true`. 

The weird thing is that we had apparently default it to `false` for `calypso-build` before:

https://github.com/Automattic/wp-calypso/blob/7a493595bdb8e61fa19ec1e2f6ee41a202139f0d/packages/calypso-build/webpack.config.js#L110

I found that weird while working on this PR, but tried to stick to the previous behavior as much as possible. I'm however fine to switch to `postcss-custom-properties`' default, as it seems to make sense.

> You can just leave the option out and specify just
> 
> ```
> 'postcss-custom-properties': {}
> ```

Okay, will do :+1:

> Then the plugin will insert polyfill fallbacks for IE11 whenever it can. See also my reply to @simison ([#36471 (comment)](https://github.com/Automattic/wp-calypso/pull/36471#discussion_r338940454)) and the PR description of #36471. There are details about what the `postcss-custom-properties` plugin does.

:ok_hand:

Reply:
4f1fea79e7